images/influxdb/run.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -m
+
+echo "Starting InfluxDB in the background"
+exec /influxdb/influxd -config /etc/influxdb.toml &
```

Comment:
To be frank, I've stolen this approach from Grafana's `run.sh`. I can add initialization to the watcher.